blockdag/reachability_test.go

```diff
@@ -491,23 +491,25 @@ func TestSplitFractionErrors(t *testing.T) {
 	_, _, err := interval.splitFraction(-0.5)
 	if err == nil {
 		t.Fatalf("TestSplitFractionErrors: splitFraction unexpectedly " +
-			"didn't return an error")
+			"didn't return an error for a negative fraction")
 	}
 	expectedErrSubstring := "fraction must be between 0 and 1"
 	if !strings.Contains(err.Error(), expectedErrSubstring) {
-		t.Fatalf("TestSplitFractionErrors: splitFraction returned wrong error. "+
+		t.Fatalf("TestSplitFractionErrors: splitFraction returned wrong error "+
+			"for a negative fraction. "+
 			"Want: %s, got: %s", expectedErrSubstring, err)
 	}
 
 	// Fraction > 1
 	_, _, err = interval.splitFraction(1.5)
 	if err == nil {
 		t.Fatalf("TestSplitFractionErrors: splitFraction unexpectedly " +
-			"didn't return an error")
+			"didn't return an error for a >1 fraction")
```

Comment:
`>1` without the first operand  is confusing. Please write `for a fraction>1` or `for a fraction bigger than one`